src/main/java/org/myrobotlab/service/Servo.java

```diff
@@ -126,6 +126,11 @@ protected boolean processMove(Double newPos, boolean blocking, Long timeoutMs) {
      * </pre>
      *
      */
+    // FIXME - poor implementation - should addListener(sync)
+    // and use pub/sub :(
+    for(String syncServo: syncedServos) {
+      send(syncServo, "moveTo", newPos);

```

Comment:
There should be some test to make sure a servo isn't sync'd to itself.. otherwise, you'll get into a nasty loop here.

Reply:
fixed